graphs/possible_bipartition.py

```diff
@@ -1,12 +1,49 @@
 # Can be used for BFS
-from collections import deque 
+from collections import deque
+
 
 def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n log n)
+        Space Complexity: O(n)
     """
```

Comment:
👍 Although I think the time complexity is O(NE) where n is the number of nodes (dogs) and E is the number of edges (dislikes between dogs).